docs/src/App.tsx

```diff
@@ -1,24 +1,25 @@
-import * as React from 'react';
 import { Header } from './Header';
-import { BrowserRouter as Router, Route, Link } from 'react-router-dom';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import { Docs } from './Docs';
 import { Home } from './Home';
+import { useEffect } from 'react';
 
-export class App extends React.PureComponent {
-  componentDidMount() {
-    window.addEventListener('onpopstate', e => {
+export function App() {
+  useEffect(()=>{
+    window.addEventListener('popstate', e => {
       console.log(e);
     });
-  }
-  render() {
-    return (
-      <Router>
-        <div className="wrapper">
-          <Header />
-          <Route exact path="/" component={Home} />
-          <Route path="/docs" component={Docs} />
-        </div>
-      </Router>
-    );
-  }
+  },[])
```

Comment:
This is most likely a debug print that was left behind, we can drop it

Reply:
Similar code existed in the old code, so I implemented it to keep compatibility. If you don't mind, I will try to remove it!

Reply:
https://github.com/remeda/remeda/pull/359/commits/77377cf9c944e8cdf42878bb00135abc1564e783

deleted